service/validator_options.go

```
@@ -10,6 +10,10 @@ import (
 // ValidatorOption ...
 type ValidatorOption func(c *ValidatorConfig)
 
+var defaultExternalErrorHandler = func(err error) error {
+	return err
```

this error is not handled anywhere, also one of my comment about the above ☝️